kong/plugins/azure-functions/handler.lua

```diff
@@ -54,7 +54,11 @@ function azure:access(config)
   local request_headers = kong.request.get_headers()
   local request_args = kong.request.get_query()
 
-  local upstream_uri = var.upstream_uri
+  -- strip any query args
+  local upstream_uri = var.upstream_uri or var.request_uri
+  local s, _ = string.find(upstream_uri, "?")
```

Comment:
here `_` is unnecessary,  just `local s = string.find(...)`.

Also, use `string.find(x, '?', 1, true)` to speed up.